app/ledger.js

```diff
@@ -132,7 +132,7 @@ var boot = () => {
   if ((bootP) || (client)) return
 
   bootP = true
-  fs.access(statePath, fs.FF_OK, (err) => {
+  fs.access(pathName(statePath), fs.FF_OK, (err) => {
```

Comment:
pathName could probably just be called only once again above instead of repeatedly below, downside is that someone could forget to call it when using statePath (which isn't actually a path) later. I'll merge though.